src/gc/impl/conservative/gc.d

```diff
@@ -56,6 +57,11 @@ else                   import core.stdc.stdio : sprintf, printf; // needed to ou
 import core.time;
 alias currTime = MonoTime.currTime;
 
+extern (C) nothrow @nogc
+{
+    void _Exit(int);
+}
+
```

Comment:
You don't need braces for a single definition. Doesn't `_Exit` already exist in some core.sys.... module? You should better import that where used then.